students/halas_maciek/lesson_01/lesson_03-2.py

```diff
@@ -0,0 +1,22 @@
+def collatz(number):
+    collatzed_number = 0
+    if number % 2 == 0:
```

Comment:
You can use sthi like that:
return number // 2 if number % 2 == 0 else 3 * number + 1

It's only tip because Your solution is propably more readable